### PR TITLE
Saved prelude-manager-oss iptables rules after reload

### PR DESCRIPTION
**Pull request by agil3b3ast**
Used iptables-services to save iptables rules

We use only iptables and not firewalld

### Description of the Change

Saved prelude-manager-oss iptables rules after reload.

### Alternate Designs

N/A

### Benefits

After this, _prelude-manager_'s iptables rules persist even if `vagrant reload prelude-manager-oss` is executed. 
### Possible Drawbacks

N/A

### Applicable Issues

#38

### CODE REVIEW

**Comment by agil3b3ast** [comment]
@ferrarimarco i did not understand your request, what should i keep in this pull request?

File: provisioning/prelude-manager-oss/install-packages.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -exe
 
 dnf install -y \
+  iptables-services \
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/install-packages.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 2]
```diff
@@ -448,7 +461,18 @@ EOF
 
 systemctl enable prelude-registrator
 
+systemctl mask firewalld.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 3]
```diff
@@ -448,7 +461,18 @@ EOF
...
+systemctl enable iptables.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 4]
```diff
@@ -448,7 +461,18 @@ EOF
...
+systemctl enable ip6tables.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 5]
```diff
@@ -448,7 +461,18 @@ EOF
...
+
 iptables -I INPUT -p tcp -s 0.0.0.0/0 --dport 4690 -j ACCEPT
 iptables -I INPUT -p tcp -s 0.0.0.0/0 --dport 5553 -j ACCEPT
 
-mkdir -p /var/run/prelude-manager
+#saving iptables rules
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 6]
```diff
@@ -448,7 +461,18 @@ EOF
...
+iptables-save > /etc/sysconfig/iptables
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 7]
```diff
@@ -448,7 +461,18 @@ EOF
...
+
+systemctl stop firewalld.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 8]
```diff
@@ -448,7 +461,18 @@ EOF
...
+systemctl start iptables.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

File: provisioning/prelude-manager-oss/post-install.sh [hunk 9]
```diff
@@ -448,7 +461,18 @@ EOF
...
+systemctl start ip6tables.service
```
Comment (ferrarimarco) [inline provisioning/prelude-manager-oss/post-install.sh]: move this to another pull request

Comment (agil3b3ast) [reply provisioning/prelude-manager-oss/post-install.sh]: I don't understand do you want me to close this pull request and create another one? This is the fix, without it the other code is useless. The other code is simply specifying that a service should be started before another service. If you want @ferrarimarco i can create another issue to put the rest of the code but without the lines you requested to move it will not work.

**Comment by ferrarimarco** [comment]
You are addressing two separate issues here:
1. Save iptables rules and restore them when needed
1. Start prelude-manager as a service

According to me these two aspects should be dealt with in separate PRs.
So one to manage iptables and one to configure the prelude-manager service.

After reading your comment I have a clearer picture. Maybe it's better to keep iptables bits here and create a separate issue and PR for the prelude-manager service configure.

Thoughts?

**Comment by agil3b3ast** [comment]
Ok @ferrarimarco i will add the rest of the code in #41 . Actually prelude-manager is still running as a service, but running before _maria-db_ is necessary to be resilient to reboots.

**Comment by ferrarimarco** [comment]
LGTM

This is the output after a `vagrant reload prelude-manager-oss`

```
[vagrant@prelude-manager-oss ~]$ sudo iptables -S
-P INPUT ACCEPT
-P FORWARD ACCEPT
-P OUTPUT ACCEPT
-A INPUT -p tcp -m tcp --dport 5553 -j ACCEPT
-A INPUT -p tcp -m tcp --dport 4690 -j ACCEPT
```
